lazy_tensor_core/lazy_tensor_core/csrc/lazy_graph_executor.cpp

```diff
@@ -506,38 +506,44 @@ torch::lazy::Value LazyGraphExecutor::GetDeviceDataIrValue(
   return torch::lazy::MakeNode<ir::ops::DeviceData>(std::move(data));
 }
 
+torch::lazy::Value LazyGraphExecutor::GetIrValueForScalarFromCodegen (const at::Scalar& value) {
```

Comment:
we really need a better name here, FromCodegen is strictly circumstantial today and it could be used from outside of codegen, the name also doesn't reflect what's different about these scalars vs the ones from the other API.

we ran into issues with updating the other GetIrValueForScalar functions, so we'll make more complete changes in a separate PR to not couple cosmetic with functional changes.

Can you at least add a TODO comment above this func that 
1) mentions renaming it
2) explains what it does differently from the existing GetIrValueForScalar functions?